cdqa/reader/bertqa_sklearn.py

```diff
@@ -856,20 +857,23 @@ def __init__(self,
         self.doc_stride = doc_stride
         self.max_query_length = max_query_length
         self.verbose = verbose
+        self.tokenizer = tokenizer
 
     def fit(self, X, y=None):
         return self
 
     def transform(self, X):
-
-        tokenizer = BertTokenizer.from_pretrained(self.bert_model, do_lower_case=self.do_lower_case)
+        if self.tokenizer is None:
+            self.tokenizer = BertTokenizer.from_pretrained(self.bert_model, do_lower_case=self.do_lower_case)
+        else:
+            logger.info("loading tokenizer with custom vocabulary")
```

Comment:
Maybe it would be proper if you move the block to the `__init__()` method?
(I know we haven't originally done like that, but seeing the code now I think it's better)

Reply:
Thanks! Yeah I agree! I've made the commit.